docs/things-endpoint.md

```diff
@@ -206,6 +208,9 @@ To obtain an new User Token by exchanging a Refresh Token, perform an HTTP POST
     ```
 
 1. Install and configure AM as described in the [IoT evaluation guide](https://backstage.forgerock.com/docs/iot/7.1/evaluation-guide/before-you-start.html#install-am).
+1. Modify the IoT OAuth 2.0 Client `forgerock-iot-oauth2-client`:
+    * Add `publish` scope to `Scope(s)`.
+    * Add the `Refresh Token` grant to `Grant Types`.
```

Comment:
As far as I can see the refresh token grant type is already added by the IoT Service in AM, so if the IoT evaluation guide was followed then this step should not be required.

Reply:
Great spot. Furthermore, the `publish` scope is added in the evaluation guide so this section can be removed. I'll also update the video.